packages/documentation/src/stories/components/collapsible/collapsible.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import { useArgs } from '@storybook/client-api';
+import React, { MouseEventHandler } from 'react';
+import { Story, Args, StoryContext, ReactFramework } from '@storybook/react';
+import { PostCollapsible } from '@swisspost/design-system-components-react';
+import parse from 'html-react-parser';
+import { definedProperties } from '../../../utils';
+import docsPage from './collapsible.docs.mdx';
+
+export default {
+  title: 'Components/Collapsible',
+  component: PostCollapsible.displayName,
+  parameters: {
+    docs: {
+      page: docsPage,
+    },
+    controls: {
+      exclude: [
+        'Content',
+      ],
+    },
+  },
+  args: {
+    content: `<span slot="header">Titulum</span><p>Contentus momentus vero siteos et accusam iretea et justo.</p>`,
+  },
+  argTypes: {
+    content: {
+      name: 'Content',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Content',
+      },
+    },
+  },
+};
```

Comment:
Please visit the docs https://storybook.js.org/docs/react/writing-stories/introduction to ensure the correct setup here.
 The default export of a stories file, should be of type `Meta` or `ComponentMeta<typeof component>`.
 
 But as you only use the `displayName` for the `component` property, I'm not sure how it must be written at this point.